core/src/main/java/com/sequenceiq/cloudbreak/service/upgrade/rds/RdsUpgradeOrchestratorService.java

```diff
@@ -82,21 +82,21 @@ public class RdsUpgradeOrchestratorService {
     private double backupValidationRatio;
 
     public void backupRdsData(Long stackId) throws CloudbreakOrchestratorException {
-        OrchestratorStateParams stateParams = createStateParams(stackId, BACKUP_STATE);
+        OrchestratorStateParams stateParams = createStateParams(stackId, BACKUP_STATE, true);
         stateParams.setStateParams(backupRestoreEmbeddedDBStateParamsProvider.createParamsForBackupRestore());
         LOGGER.debug("Calling backupRdsData with state params '{}'", stateParams);
         hostOrchestrator.runOrchestratorState(stateParams);
     }
 
     public void restoreRdsData(Long stackId) throws CloudbreakOrchestratorException {
-        OrchestratorStateParams stateParams = createStateParams(stackId, RESTORE_STATE);
+        OrchestratorStateParams stateParams = createStateParams(stackId, RESTORE_STATE, true);
         stateParams.setStateParams(backupRestoreEmbeddedDBStateParamsProvider.createParamsForBackupRestore());
         LOGGER.debug("Calling restoreRdsData with state params '{}'", stateParams);
         hostOrchestrator.runOrchestratorState(stateParams);
     }
 
     public void installPostgresPackages(Long stackId) throws CloudbreakOrchestratorException {
-        OrchestratorStateParams stateParams = createStateParams(stackId, PG11_INSTALL_STATE);
+        OrchestratorStateParams stateParams = createStateParams(stackId, PG11_INSTALL_STATE, false);
```

Comment:
sanity check: was this intentionally set to run not just on the primary gateway?

Reply:
Yes, it was